scripts/workspace_container_test.py

```diff
@@ -0,0 +1,82 @@
+from lib.biokbase.workspace.client import Workspace
+import pytest
+import requests
+import json
+
+""" workspace_container_test.py
+
+Very simple tests to ensure that local workspace and auth2 servers are functioning correctly.
+Requires the python libraries `pytest` and `requests` to be installed.
+
+To run:
+- move this script to the `workspace_deluxe` (i.e. the parent) directory
```

Comment:
Why require the move rather than run in place?

Reply:
There's some fiddling required due to the location of the `biokbase` modules and this script. I've changed this so it now recommends running the wrapper script instead, which sidesteps all that nastiness.

Reply:
:+1: